README.md

```diff
@@ -135,6 +135,34 @@ val yaml = Color("CadetBlue", 95, 158, 160).toYaml
 val color = yaml.convertTo[Color]
 ```
 
+Example with inner objects:
```

Comment:
I wouldn't call this "Example with inner objects". I think it's more of a "Example combining custom YamlFormats", as the example with case classes already is an example with inner objects, but whose formats are provided in `DefaultYamlProtocol`.